server/src/test/java/io/crate/integrationtests/SysClusterSettingsTest.java

```diff
@@ -198,6 +211,21 @@ public void test_set_cluster_info_update_interval() throws Exception {
         assertThat(f1.get(clusterService)).isEqualTo(TimeValue.timeValueSeconds(45));
     }
 
+    @Test
+    public void test_archived_setting_can_be_reset() throws Exception {
+        execute("set global transient \"archived.cluster.routing.allocation.disk.watermark.low\"=\"70%\"");
+        var clusterService = cluster().getMasterNodeInstance(ClusterService.class);
+        assertThat(clusterService.getClusterSettings().get("archived.cluster.routing.allocation.disk.watermark.low"))
+            .isNotNull();
+        assertThat(clusterService.state().metadata().settings().get("archived.cluster.routing.allocation.disk.watermark.low"))
+            .isEqualTo("70%");
+
+        execute("reset global \"archived.cluster.routing.allocation.disk.watermark.low\"");
+        clusterService = cluster().getMasterNodeInstance(ClusterService.class);
+        assertThat(clusterService.state().metadata().settings().get("archived.cluster.routing.allocation.disk.watermark.low"))
+            .isNull();
+    }
```

Comment:
AFAIK, we want to reset `archived.` in order to be able to use SET again. 

Maybe we can assert that SET of any setting works after resetting invalid archived.